libs/designsystem/modal/src/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -648,3 +648,61 @@ describe('ModalWrapperComponent', () => {
     });
   });
 });
+
+describe('initializeContentScrollListening', () => {
```

Comment:
Consider adding this describe block to the top level describe block to avoid duplicate setup